stack.yaml

```diff
@@ -32,14 +32,20 @@ packages:
 - smos-github
 - smos-jobhunt
 
-- smos-stripe-client
+# - smos-stripe-client
 
 # These are duplicated in nix/overlay.
 ghc-options:
   "$locals": -Wall -Wincomplete-uni-patterns -Wincomplete-record-updates -Wpartial-fields -Widentities -Wredundant-constraints -Wcpp-undef -O0 -Wunused-packages
   smos-stripe-client: -w # Turn off warnings for the generated code.
 
 extra-deps:
+- vty-6.2@sha256:3536dc83a3fee17d9a114baf58fe47b6f080c24987266f0cd0b7b4b1fcd9cf19,3520
```

Comment:
@NorfairKing   crossplatform version requires the updated packages as indicated.